modApiServer/test/org/aion/api/server/TxRecptLgTest.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *
+ *     This file is part of the aion network project.
+ *
+ *     The aion network project is free software: you can redistribute it
+ *     and/or modify it under the terms of the GNU General Public License
+ *     as published by the Free Software Foundation, either version 3 of
+ *     the License, or any later version.
+ *
+ *     The aion network project is distributed in the hope that it will
+ *     be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ */
+package org.aion.api.server;
+
+import static junit.framework.TestCase.assertEquals;
+import java.math.BigInteger;
+import java.util.List;
+import org.aion.api.server.types.TxRecptLg;
+import org.aion.base.util.ByteUtil;
+import org.aion.crypto.ECKey;
+import org.aion.mcf.core.ImportResult;
+import org.aion.mcf.vm.types.DataWord;
+import org.aion.mcf.vm.types.Log;
+import org.aion.zero.impl.BlockContext;
+import org.aion.zero.impl.StandaloneBlockchain;
+import org.aion.zero.impl.types.AionTxInfo;
+import org.aion.zero.types.AionTransaction;
+import org.aion.zero.types.AionTxReceipt;
+import org.junit.Assert;
+import org.junit.Test;
+import org.aion.base.type.Address;
+
+
+
+public class TxRecptLgTest {
+
+/*
+pragma solidity ^0.4.0;
+
+contract A{
+    event AE(uint128 value);
+    event AEA(uint128 value);
+    event AEB(uint128 value);
+
+    function AA(address addr){
+        AE(1);
+        AEA(2);
+        //B(addr).BB(addr, 1, 256, 128)
+        addr.call.value(0)(bytes4(sha3("BB(address,uint128,bytes32,bytes32)")), addr, 1, 256, 128);
+        AEB(3);
+    }
+}
+
+contract B{
+    event BE(address indexed _recipient, uint128 indexed _amount, bytes32 indexed _foreignNetworkId, bytes32 _foreignData);
+
+    function BB(address _recipient, uint128 _amount, bytes32 _foreignNetworkId, bytes32 _foreignData){
+        BE(_recipient, _amount, _foreignNetworkId, _foreignData);
+    }
+}
+*/
+
+    @Test
+    public void TestTxRecptLg() throws InterruptedException {
+        System.out.println("run TestTxRecptLg.");
```

Comment:
I don't think it's necessary to print the test name (I believe some old tests did this, but I think we're trying to phase it out).

Reply:
agreed,

Reply:
removed